uisdk/src/main/java/com/karhoo/uisdk/KarhooUISDKConfiguration.kt

```diff
@@ -11,6 +11,8 @@ interface KarhooUISDKConfiguration : KarhooSDKConfiguration, PaymentProviderConf
 
     fun bookingMetadata(): HashMap<String, String>? = null
 
+    fun useCheckboxOnTermsAndConditions(): Boolean = false
+
```

Comment:
If I correctly understand this parameter is set by DP? If that's true it will be great to name it in the same way in both platforms. in iOS it's it is named: isExplicitTermsAndConfitionsAprovalRequired. Don't know which name is better...